src/sentry/static/sentry/app/utils/theme.tsx

```diff
@@ -76,6 +76,43 @@ const colors = {
   placeholderBackground: '#f5f5f5',
 };
 
+// from the Sentry design system: most useful for showing a transition
+// from one extreme, through a neutral middle, to an opposite extreme.
+export const divergentColorScale = {
+  blue: '#444674',
+  maroon: '#955389',
+  magenta: '#C15584',
+  salmon: '#E65D73',
+  purple: '#694E86',
+  orange: '#F27A58',
+  marigold: '#F6983B',
+  yellow: '#F2B712',
+  softBlue: '#535577',
+  softMaroon: '#805567',
+  softMagenta: '#70506A',
+  softSalmon: '#96545E',
+  softPurple: '#6B597E',
+  softOrange: '#D17D65',
+  softMarigold: '#E4944E',
+  softYellow: '#EDC658',
+  darkOrange: '#98361B',
+  darkBlue: '#1E1F33',
+  darkMaroon: '#382947',
+  darkMagenta: '#522E4B',
+  darkSalmon: '#833054',
+  darkPurple: '#AF2C41',
+  darkMarigold: '#C36609',
+  darkYellow: '#E2B22E',
+};
+
+// you can link span operation types to specific colors here
+export const spanColors = {
+  default: divergentColorScale.blue,
+  transaction: divergentColorScale.softBlue,
+  db: divergentColorScale.softPurple,
+  http: divergentColorScale.marigold,
```

Comment:
:+1: This is a nice feature to have if we need to tweak common span ops later.

Reply:
yeah we'll probably wanna add some more stuff here. the random part is cool, but nothing beats handpicked options, because we can create complimentary combinations between common pairs.